buildSrc/src/main/java/com/google/firebase/gradle/plugins/FirebaseLibraryPlugin.kt

```diff
@@ -54,7 +54,6 @@ class FirebaseLibraryPlugin : BaseFirebaseLibraryPlugin() {
 
     project.apply<DackkaPlugin>()
     project.apply<GitSubmodulePlugin>()
-    project.apply<PostReleasePlugin>()
```

Comment:
I'm unsure I understand why this deletion is ok. 

Is the plugin just being applied differently? Is this offset by the change in PublishingPlugin.kt:98?

Reply:
> Is this offset by the change in PublishingPlugin.kt:98?

Exactly. Perviously, `PostReleasePlugin` was applied to every firebase lib. Now, it only gets applied to releasing libs.